src/metabase/integrations/slack.clj

```diff
@@ -105,7 +115,7 @@
   "Calls Slack API `conversations.list` and returns list of available 'conversations' (channels and direct messages). By
   default only fetches channels."
   [& {:as query-parameters}]
-  (let [params (merge {:exclude_archived true, :types "public_channel,private_channel"} query-parameters)]
+  (let [params (merge {:exclude_archived true, :types "public_channel"} query-parameters)]
```

Comment:
Removed the `private_channel` read since it requires an additional scope, and I don't think it's necessary.

Reply:
So is it still possible to send to private channel? That was a very used functionality #2694

Reply:
@flamber just verified that sending subscriptions to private channels still works for Slack apps, _if_ a user has added the app to the channel (via `/invite @AppName`).

The channel name has to still be edited in the DB, though, so it's pretty hacky. Should we reopen that issue so that we can eventually provide a more official way to do this? (e.g. a field on the subscription sidebar to type a private channel name, since won't doesn't show up in the dropdown)

The other issue is that the charts still need to be uploaded to a public `metabase_files` channel, even if the subscription is sent to a private channel. So we shouldn't give the impression that the information in the charts is private.